lib/gateway/actionParams.js

```diff
@@ -17,7 +17,7 @@ ActionParams.prototype.getCommonAuthCallback = function (req, res, next) {
       return next(err);
     }
     if (user) {
-      req.logIn(user, (err) => next(err));
+      req.logIn(user, this, (err) => next(err));
```

Comment:
Can we just pass `{session: this.session}` so we do not pass the whole object?

Now it is just that parameter, but just in case...

Reply:
passport has it as object. so I assume they can at some point add other params. in this case passing only session will require code modification. in current version any new params is passed.